webextension/css/popup.css

```diff
@@ -1,16 +1,55 @@
 /* General Rules and Resets */
-body {
-  inline-size: 300px;
-  max-inline-size: 300px;
+* {
+  font-size: inherit;
+  margin-block-end: 0;
+  margin-block-start: 0;
+  margin-inline-end: 0;
+  margin-inline-start: 0;
+  padding-block-end: 0;
+  padding-block-start: 0;
+  padding-inline-end: 0;
+  padding-inline-start: 0;
```

Comment:
The pop-up margins & paddings become very small and tight, which condenses all the hit-targets ...

<img width="302" alt="image" src="https://user-images.githubusercontent.com/71928/26837480-5e3df8aa-4a92-11e7-82cd-e8baeac05c52.png">

<img width="302" alt="image" src="https://user-images.githubusercontent.com/71928/26837526-77ca25e6-4a92-11e7-8057-589e570292d9.png">

<img width="302" alt="image" src="https://user-images.githubusercontent.com/71928/26837547-88713f10-4a92-11e7-9581-8bced484e301.png">